src/Sylius/Bundle/AdminBundle/Resources/views/Product/_showInShopButton.html.twig

```diff
@@ -1,37 +1,36 @@
-                                <i class="angle right icon"></i>
-                                {{ 'sylius.ui.show_in'|trans }}
-                                {{ channel.name }} ({{ channel.code }})
-                            </a>
-                        {% endfor %}
+            {% if enabledChannels|length > 1 %}
+                <div class="ui floating dropdown labeled icon button">
+                    <i class="share alternate icon"></i>
+                    <span class="text">
+                        {{ 'sylius.ui.show_product_in_shop_page'|trans }}
+                    </span>
+                    <div class="menu">
+                        <div class="scrolling menu">
+                            {% for channel in enabledChannels %}
+                                {% set url = channel.hostname is not null ? 'http://' ~ channel.hostname ~ path('sylius_shop_product_show', {'slug': product.slug, '_locale': channel.defaultLocale.code}) : url('sylius_shop_product_show', {'slug': product.slug, '_locale': channel.defaultLocale.code}) %}
+                                <a href="{{ url|raw }}" class="item">
+                                    <i class="angle right icon"></i>
+                                    {{ 'sylius.ui.show_in'|trans }}
+                                    {{ channel.name }} ({{ channel.code }})
+                                </a>
+                            {% endfor %}
+                        </div>
                     </div>
                 </div>
-            </div>
-        </div>
-    </div>
-{% elseif enabledChannels|length == 1 %}
-    <div class="middle aligned column">
-        <div class="ui right floated buttons">
-            {% for channel in enabledChannels %}
-                {% set url = channel.hostname is not null ? 'http://' ~ channel.hostname ~ path('sylius_shop_product_show', {'slug': product.slug, '_locale': channel.defaultLocale.code}) : url('sylius_shop_product_show', {'slug': product.slug, '_locale': channel.defaultLocale.code}) %}
-                <a class="ui labeled icon button" href="{{ url|raw }}">
-                    <i class="angle right icon"></i>
-                    {{ 'sylius.ui.show_product_in_shop_page'|trans }}
-                </a>
-            {% endfor %}
+            {% else %}
+                {% for channel in enabledChannels %}
+                    {% set url = channel.hostname is not null ? 'http://' ~ channel.hostname ~ path('sylius_shop_product_show', {'slug': product.slug, '_locale': channel.defaultLocale.code}) : url('sylius_shop_product_show', {'slug': product.slug, '_locale': channel.defaultLocale.code}) %}
+                    <a class="ui labeled icon button" href="{{ url|raw }}">
+                        <i class="angle right icon"></i>
+                        {{ 'sylius.ui.show_product_in_shop_page'|trans }}
+                    </a>
+                {% endfor %}
+            {% endif %}
         </div>
```

Comment:
> When a product is disabled the "Show in shop" button should be disabled too  <- task
>Hide show in shop button when product is disabled <- your PR
IMO you should disable button if the product is disabled, not hide him.
WDYT @GSadee  @CoderMaggie